scripts/battlefields/Cloister_of_Tremors/trial_by_earth.lua

```diff
@@ -0,0 +1,38 @@
+-----------------------------------
+-- Area: Cloister of Tremors
+-- BCNM: Trial by Earth
+-----------------------------------
+local cloisterOfTremorsID = zones[xi.zone.CLOISTER_OF_TREMORS]
+-----------------------------------
+
+local content = BattlefieldQuest:new({
+    zoneId           = xi.zone.CLOISTER_OF_TREMORS,
+    battlefieldId    = xi.battlefield.id.TRIAL_BY_EARTH,
+    maxPlayers       = 6,
+    timeLimit        = utils.minutes(30),
+    index            = 0,
+    entryNpc         = 'EP_Entrance',
+    exitNpc          = 'Earth_Protocrystal',
+    requiredKeyItems = { xi.ki.TUNING_FORK_OF_EARTH },
+
+    questArea = xi.questLog.BASTOK,
+    quest     = xi.quest.id.outlands.TRIAL_BY_EARTH,
+})
+
```

Comment:
I think this is missing a function:  `function content:onEventFinishWin(player, csid, option, npc`

Reply:
Its not.  That's the only Trial covered by interaction for the quest

Reply:
Ahh ok, I did not realize that. Looks good then! Huge thanks for doing this 😄